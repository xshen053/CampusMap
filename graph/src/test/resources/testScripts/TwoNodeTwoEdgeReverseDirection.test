# Create a graph
CreateGraph TNOERD_graph

# Add two nodes
AddNode TNOERD_graph b
AddNode TNOERD_graph a

# Add two edges
AddEdge TNOERD_graph a b AB
AddEdge TNOERD_graph b a BA

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNOERD_graph
ListChildren TNOERD_graph a
ListChildren TNOERD_graph b