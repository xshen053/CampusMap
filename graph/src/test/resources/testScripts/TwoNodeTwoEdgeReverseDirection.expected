# Create a graph
created graph TNOERD_graph

# Add two nodes
added node b to TNOERD_graph
added node a to TNOERD_graph

# Add two edges
added edge AB from a to b in TNOERD_graph
added edge BA from b to a in TNOERD_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNOERD_graph contains: a b
the children of a in TNOERD_graph are: b(AB)
the children of b in TNOERD_graph are: a(BA)